client/parser/variables_test.go

```diff
@@ -16,53 +17,37 @@ func variableReference(name string) model.VariableReference {
 	return model.VariableReference{VariableName: name}
 }
 
+func variableNameCheck(name string) parseTestCase {
+	return parseTestCase{
+		label: fmt.Sprintf("variable definition and reference: %s", name),
+		given: fmt.Sprintf("%[1]s = r \n%[1]s", name),
```

Comment:
TIL that you could do this. Mind blown.

Reply:
Haha I had to search it up too. Pretty neat though.